lms/djangoapps/bulk_email/models.py

```diff
@@ -454,7 +454,7 @@ def __unicode__(self):
 # .. toggle_name: require_course_email_auth
 # .. toggle_type: ConfigurationModel
 # .. toggle_default: True (enabled)
-# .. toggle_description: If the flag is enabled, course-specific authorization is required, and the course_id is either not provided or not authorixed, the feature is not available.
+# .. toggle_description: If flag is enabled, course-specific authorization is required
```

Comment:
We should not sacrifice clarity of expression for line-length.  Why don't we adjust the pylint rules instead?

Reply:
The pylint rule already excuses pii annotation lines from line-length checks:
```
ignore-long-lines = ^\s*(# )?((<?https?://\S+>?)|(\.\. pii: .*))$
```
We could amend it to be also excuse any annotation line:
```
ignore-long-lines = ^\s*(# )?((<?https?://\S+>?)|(\.\. pii: .*)|(# .. \w+: .*))$
```
@pwnage101 @doctoryes ???

Reply:
Fine with me if it works for @bmedx 😸

Reply:
Seems good to me 👍

Reply:
I'll update edx-lint, then we can pull it here.

Reply:
https://github.com/edx/edx-lint/pull/98

Reply:
#20212

Reply:
#20212 is now merged.  Rebase on that, and let those words flow!

Reply:
@nedbat Ready for re-review.  I'll do a separate PR for any updates after https://github.com/edx/edx-toggles/pull/3 is updated with Nimisha and Robert's suggestions.